docker-compose.override.yml

```
@@ -0,0 +1,38 @@
+    ports:
+      - "3306:3306"
+    networks:
+      - app-network
+    environment:
+      - SSH_KEY=/root/.ssh/id_rsa
+
+  mongo-tunnel:
+    container_name: mongo-tunnel
+    image: alpine:latest
+    command: >
+      sh -c "apk add --no-cache openssh && 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)